BackendApartmentReservation/BackendApartmentReservation/Program.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+
+namespace BackendApartmentReservation
+{
+    public class Program

```

Comment:
### Comment 
Static code analysis suggests that "Utility classes should not have public constructors". 
Maybe we can add 'static' keyword to the class declaration or  add a protected constructor? 
Ignore this if not applicable. 😄

Reply:
I saw that :D its an auto generated code, so I am not sure if its really applicable here :D